src/explore-education-statistics-admin/src/pages/admin-dashboard/components/__tests__/MethodologySummary.test.tsx

```diff
@@ -825,6 +825,102 @@ describe('MethodologySummary', () => {
     );
   });
 
+  describe('Removing an external methodology', () => {
+    test('shows the confirm modal when clicking the Remove button', async () => {
+      render(
+        <MemoryRouter>
+          <MethodologySummary
+            publication={testPublicationWithExternalMethodology}
+            topicId={testTopicId}
+            onChangePublication={noop}
+          />
+        </MemoryRouter>,
+      );
+
+      userEvent.click(
+        screen.getByTestId(
+          'Expand Details Section Ext methodolology title (External)',
+        ),
+      );
+
+      expect(
+        screen.getByRole('button', { name: 'Remove external methodology' }),
+      ).toBeInTheDocument();
+
+      userEvent.click(
+        screen.getByRole('button', { name: 'Remove external methodology' }),
+      );
+
+      await waitFor(() => {
+        expect(
+          screen.getByRole('heading', { name: 'Remove external methodology' }),
+        ).toBeInTheDocument();
+
+        expect(
+          screen.getByText(
+            'Are you sure you want to remove this external methodology?',
+          ),
+        ).toBeInTheDocument();
+
+        expect(
+          screen.getByRole('button', { name: 'Confirm' }),
+        ).toBeInTheDocument();
+
+        expect(
+          screen.getByRole('button', { name: 'Cancel' }),
+        ).toBeInTheDocument();
+      });
+    });
+
+    test('calls the service to remove the Methodology when the confirm button is clicked', async () => {
+      render(
+        <MemoryRouter>
+          <MethodologySummary
+            publication={testPublicationWithExternalMethodology}
+            topicId={testTopicId}
+            onChangePublication={noop}
+          />
+        </MemoryRouter>,
+      );
+
+      userEvent.click(
+        screen.getByTestId(
+          'Expand Details Section Ext methodolology title (External)',
+        ),
+      );
+
+      userEvent.click(
+        screen.getByRole('button', { name: 'Remove external methodology' }),
+      );
+
+      await waitFor(() => {
+        expect(
+          screen.getByRole('heading', { name: 'Remove external methodology' }),
+        ).toBeInTheDocument();
+      });
+
+      userEvent.click(screen.getByRole('button', { name: 'Confirm' }));
+
+      const updatedPublication = {
+        title: testPublicationWithExternalMethodology.title,
+        contact: {
+          contactName: testContact.contactName,
+          contactTelNo: testContact.contactTelNo,
+          teamEmail: testContact.teamEmail,
+          teamName: testContact.teamName,
+        },
+        topicId: testTopicId,
+      };
+
+      await waitFor(() => {
+        expect(publicationService.updatePublication).toHaveBeenCalledWith(
```

Comment:
Little tip for checking that the types for the arguments passed to `toHaveBeenCalledWith` are correct:

```ts
expect(publicationService.updatePublication).toHaveBeenCalledWith<
  Parameters<typeof publicationService.updatePublication>>(
    testPublicationWithExternalMethodology.id,
    updatedPublication
  );
```

It's not 100% necessary, but it can help get with getting earlier feedback that there is something wrong.